packages/tool-bash/src/__tests__/bash.test.ts

```
@@ -0,0 +1,119 @@
+import { describe, it, expect, vi } from 'vitest';
+import {
+  HookBus,
+  makeChatContext,
+  createLogger,
+  type SandboxSpawnInput,
+  type SandboxSpawnResult,
+} from '@ax/core';
+import { createSandboxSubprocessPlugin } from '@ax/sandbox-subprocess';
```

ESLint

[error] '@ax/sandbox-subprocess' import is restricted from being used by a pattern. Cross-plugin imports are forbidden. Plugins communicate through the hook bus only. See CLAUDE.md invariant 2. The only `@ax/`* imports allowed in plugin code are `@ax/core` and `@ax/test-harness`.

(no-restricted-imports)